pkg/v1/tkg/client/package_helper.go

```diff
@@ -38,6 +38,9 @@ func GetCorePackagesFromClusterBootstrap(clusterBootstrap *runtanzuv1alpha3.Clus
 	// Core packages CNI, CSI and CPI (other than Kapp) installed in tkg-system namespace in case of tkgm, and in case of tkgs installed in vmware-system-tkg namespace
 	if clusterBootstrap.Spec.CNI != nil && clusterBootstrap.Spec.CNI.ValuesFrom != nil && clusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef != nil {
 		name := strings.TrimSuffix(clusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef.Name, suffixStr)
+		if clusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef.Kind == "AntreaConfig" {
+			name = name + "-antrea"
+		}
```

Comment:
This change is done because currently, even if the clusterboostrap shows the CNI ref name as "cc-cb", the name of the package installed is "cc-cb-antrea". This behavior needs to be investigated further.

Reply:
This is a defect in the pacakge_helper.go code. Removing this fix and filing bug